update-ngxblocker

```diff
@@ -272,16 +281,24 @@ send_email() {
 		print_message "${BOLDYELLOW}WARN${RESET}: missing mail command => ${BOLDWHITE}disabling emails${RESET}.\n\n"
 	fi
 }
+send_email_via_mailgun() {
+	echo "Mailgunning report to: ${BOLDWHITE}$EMAIL${RESET}\n\n";
+	sed -i 's/\x1b\[[0-9;]*m//g' $EMAIL_REPORT
+	report="$(cat $EMAIL_REPORT)"
+	echo 'api:$MG_API_KEY https://api.mailgun.net/v3/$MG_DOMAIN/messages -F from=$MG_FROM -F to=$EMAIL -F subject='Nginx Bad Bot Blocker Updated' -F text="$report"'
+	curl -s --user api:$MG_API_KEY https://api.mailgun.net/v3/$MG_DOMAIN/messages -F from='botblocker<'$MG_FROM'>' -F to=$EMAIL -F subject='Nginx Bad Bot Blocker Updated' -F text="$report"
+}
```

Comment:
Your `echo` of info was not double quoted & would not be expanded by a `shell`

I think this function can be better expressed as:

```
send_email_via_mailgun() {
	local api_call= endpoint="https://api.mailgun.net/v3/$MG_DOMAIN/messages"

	echo "Mailgunning report to: ${BOLDWHITE}$EMAIL${RESET}\n\n";
	sed -i 's/\x1b\[[0-9;]*m//g' $EMAIL_REPORT
	report="$(cat $EMAIL_REPORT)"

	api_call="$endpoint -F from=$MG_FROM -F to=$EMAIL -F subject='Nginx Bad Bot Blocker Updated' -F text="$report"'

	echo "api:$MG_API_KEY $api_call"
	curl -s --user api:$MG_API_KEY $api_call
}
```
then if the `mailgun` api or endpoint changes the function only has to be changed in one place not two

Reply:
Removed echo -- Didn't need. Added endpoint variable. Cleaner.